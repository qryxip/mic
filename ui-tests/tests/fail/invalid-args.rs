use mic::answer;

#[answer(+)]
fn main() -> _ {
    42
}
