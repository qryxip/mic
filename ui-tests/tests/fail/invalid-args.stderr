error: expected expression
 --> tests/fail/invalid-args.rs:3:10
  |
3 | #[answer(+)]
  |          ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/invalid-args.rs:1:1
  |
1 | / use mic::answer;
2 | |
3 | | #[answer(+)]
4 | | fn main() -> _ {
5 | |     42
6 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/fail/invalid-args.rs`
